ui/helpers/constants/settings.js

```diff
@@ -342,6 +342,13 @@ export const SETTINGS_CONSTANTS = [
     route: `${EXPERIMENTAL_ROUTE}#autodetect-nfts`,
     icon: 'fa fa-flask',
   },
+  {
+    tabMessage: (t) => t('experimental'),
+    sectionMessage: (t) => t('securityAlerts'),
```

Comment:
We also have two entries in the Experimental section that come up while searching for `security alert`? One is `Security >> Security alerts`, and the second is `Privacy >> Enable security alerts`. Should we have different labels to help differentiate while searching?

Reply:
@bschorchit Need your input on the labelling here please. Thanks.

Reply:
That's a great catch.
Is this on develop only? I don't see any `Security >> Security alerts` in prod right now. Do we know who created the code for the Security alerts one? I'll reach out to the PM on the respective team to understand the feature and figure out the renaming here.

Reply:
Or if there's a description on the settings for that, it might be enough for me to understand more about it as well.

Reply:
Ooh wait, is this the OpenSea + Blockaid feature? This will be deprecated before we release the new Blockaid integration so we can ignore this conflict for now.

Reply:
Yes this is the Opens + Blockaid settings. Thanks Barbara.

Reply:
Sounds good 👍